apps/platform/src/app/projects/projects.service.ts

```diff
@@ -68,12 +70,17 @@ export class ProjectsService {
     return response;
   }
 
-  public getProjectSimulation(id: string): Observable<any> {
+  public getProjectSimulation(id: string): Observable<any> { // SimulationRun
     const url = this.endpoints.getSimulationRunEndpoint(id);
-    const response = this.http.get<string>(url).pipe();
+    const response = this.http.get<any>(url).pipe(shareReplay(1)); // SimulationRun
     return response;
   }
 
+  public getProjectSimulationLog(id: string): Observable<any> {
+    const endpoint = `${urls.dispatchApi}logs/${id}`;
```

Comment:
The endpoint should be retrieved from the Endpoints class not constructed manually.

Reply:
Done